packages/connectivity_plus/tizen/src/connection.cc

```diff
@@ -14,6 +14,8 @@ static ConnectionType ToConnectionType(connection_type_e type) {
       return ConnectionType::kMobile;
     case CONNECTION_TYPE_ETHERNET:
       return ConnectionType::kEthernet;
+    case CONNECTION_TYPE_BT:
+      return ConnectionType::kBluetooth;
```

Comment:
Should this function return `ConnectionType::kBluetooth` if the system reported network type is `CONNECTION_TYPE_NET_PROXY`? As far as I understand the proxy type means that the wearable device is using the bluetooth-connected phone's connection.

Reply:
Relax! :smile: 
Perhaps it is a gimmick that was entered because of a some scenario in the past.